### DISCUSSION TITLE
Field "Balancing" not functional  on Victron VRM

### DISCUSSION THREAD

**Discussion by merlep** [Louisvdw/dbus-serialbattery#281] [Q&A] [answered]
HAve JK-BMS 16S connected to Venus V2.91 with dbus 0.13. During my month end balancing operation today  I manually enabled balancing for about an hour  however  there was no change in reported field  "Balancing " in the VRM downloadable csv data file. The field was populated by "InActive"

**Comment by mr-manuel** [comment, marked as answer]
This should be fixed after https://github.com/Louisvdw/dbus-serialbattery/pull/468 is merged.
